geocoding_ios/lib/geocoding_ios.dart

```diff
@@ -57,6 +57,20 @@ class GeocodingIOS extends GeocodingPlatform {
     return Placemark.fromMaps(placemarks);
   }
 
+  @override
+  Future<bool> isPresent() async {
+    try {
+      final isPresent = await _channel.invokeMethod(
+        'isPresent',
+      );
+
+      return isPresent;
+    } on PlatformException catch (e) {
+      _handlePlatformException(e);
+      rethrow;
+    }
```

Comment:
Wouldn't it be easier / faster to simply return `true` in this method and skip the whole native implementation?